tests/Unit/StoreApiTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace Tests\Unit;
+
+use Tests\TestCase;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class StoreApiTest extends TestCase
+{
+	// Test it can store new user
+    public function test_it_can_store_new_user($name ="dummy name", $address="dummy address", $age="12")
+    {
+    	$request_array = [
+	        'name' => $name,
+	        'address' => $address,
+	        'age' => $age,
+    	];
+        $response = $this->call('POST', 'api/users', $request_array);
```

Comment:
This is common process so you can make it common private method or something. Then each test cases can call the same method with different arguments.